src/sentry/static/sentry/app/views/releases/list/releaseAdoption.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {withTheme} from 'emotion-theming';
+
+import Count from 'app/components/count';
+import ProgressBar from 'app/components/progressBar';
+import TextOverflow from 'app/components/textOverflow';
+import Tooltip from 'app/components/tooltip';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Theme} from 'app/utils/theme';
+
+import {releaseDisplayLabel} from '../utils';
+
+import {DisplayOption} from './utils';
+
+type Props = {
+  adoption: number;
+  releaseCount: number;
+  projectCount: number;
+  displayOption: DisplayOption;
+  theme: Theme;
+  withLabels?: boolean;
+};
+
+function ReleaseAdoption({
+  adoption,
+  releaseCount,
+  projectCount,
+  displayOption,
+  theme,
+  withLabels,
+}: Props) {
+  return (
+    <div>
+      {withLabels && (
+        <Labels>
+          <TextOverflow>
+            <Count value={releaseCount} />/<Count value={projectCount} />{' '}
+            {releaseDisplayLabel(displayOption, projectCount)}
+          </TextOverflow>
+
+          <span>{!adoption ? 0 : adoption < 1 ? '<1' : Math.round(adoption)}%</span>
+        </Labels>
+      )}
+
+      <Tooltip
+        containerDisplayMode="block"
+        popperStyle={{
+          background: theme.gray500,
```

Comment:
Should this use a theme alias instead of a specific color for better darkmode compatibility?

Reply:
All of these are used to stay consistent with the chart tooltip which uses these very values as they look the same on both light and dark mode:
https://github.com/getsentry/sentry/blob/fb08ac6d622de2bd2d11971bcc494c760776c79f/src/sentry/static/sentry/app/components/charts/baseChart.tsx#L442

![image](https://user-images.githubusercontent.com/9060071/110941243-50af3900-8338-11eb-81ad-356288dfb848.png)

![image](https://user-images.githubusercontent.com/9060071/110941886-46da0580-8339-11eb-9abc-d386486bce26.png)

Should we create aliases anyway and refactor them both here and in chart tooltips?

Reply:
should we move `Total Project `above `This Release`?

Reply:
Yeah, I was wondering the same thing. It was designed this way but I think we can safely reorder it to increase the consistency even more 👍